docs/topics/compiler-reference.md

```diff
@@ -159,7 +159,7 @@ Use a custom JDK home directory to include into the classpath if it differs from
 ### -Xjdk-release=version
 
 Specify the target version of the generated JVM bytecode. Limit the API of the JDK in the classpath to the specified Java version. 
-Automatically sets [`-jvm-target version`](#jvm-target-version). Works only for [OpenJDK distributions](https://openjdk.java.net/projects/jdk/18/).
+Automatically sets [`-jvm-target version`](#jvm-target-version). The effect of use is [guaranteed](https://youtrack.jetbrains.com/issue/KT-29974) only for [OpenJDK distributions](https://openjdk.java.net/projects/jdk/18/).
```

Comment:
I think it's best to avoid usage of any specific JDK distributions and just write that it's not guranteed to work on all JDK disributions like suggested in https://github.com/JetBrains/kotlin-web-site/pull/2991#discussion_r894456634

Reply:
Oops, indeed, by some reason I had the page opened without the udalov's comment, sorry.